<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget3</class>
 <widget class="QWidget" name="Widget3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>中点圆算法</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中点圆算法</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>x0  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>y0  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>r  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_r">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_draw">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>320</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>绘制</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>20</width>
     <height>380</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_draw_2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>370</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>267</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>透明</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>浅灰</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>深黄</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_x0</tabstop>
  <tabstop>lineEdit_y0</tabstop>
  <tabstop>lineEdit_r</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_draw</tabstop>
  <tabstop>pushButton_draw_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
